corehq/apps/translations/app_translations/utils.py

```diff
@@ -202,6 +205,49 @@ def none_or_unicode(val):
     return rows
 
 
+def text_and_audio_changed(current_row, previous_row, headers):
```

Comment:
for the logic of this method, can we instead
1. pass the app.langs
2. compare `default_{lang}` and `audio_{lang}' in between previous_row and current_row and just return `True` if both are changed?

I believe that more accurately replicates what we want to check? 
Validate that both the audio path and the text for each language, has not changed.

Reply:
This seems like a fine suggestion but a stylistic one. How would that be more accurate?